src/core/core.scale.js

```diff
@@ -344,9 +355,12 @@ var Scale = Element.extend({
 		me.beforeFit();
 		me.fit();
 		me.afterFit();
-		//
+		// Auto-skip
+		me._ticksToDraw = tickOpts.display && tickOpts.autoSkip ? me._autoSkip(me.getTicks()) : ticks;
+
 		me.afterUpdate();
 
+		// TODO(v3): remove minSize as a public property and return value from all layout boxes. It is unused
```

Comment:
All of them I think: `me.min/maxSize`, `me.min/maxWidth`, `me.min/maxHeight`, `me.margins` :) (not sure which one are currently exposed).

Reply:
I've updated the comment to include `maxWidth` and `maxHeight`. The other properties you mentioned do not exist